router.md

```diff
@@ -20,8 +20,31 @@ Like Fritz!Box, many routers offer a built-in DHCP server. To use this setup, en
 
 Now, add public DNS servers to the router's settings under `Internet/Zugangsdaten/DNS-Server`.
 
-In Pi-hole, only the Fritz!Box should be added as an upstream server. Check `Use Conditional Forwarding` and type in the following values:
+## Using DNS over TLS (DoT)
 
+DNS requests from the local network should be encrypted via TLS before being sent to an upstream DNS server. This can be configured in the router under `Internet/Zugangsart/DNS-Server`. Here are some as an example:
```

Comment:
```suggestion
*Note: This technology is rather unstable. It may cause complete DNS outage if encryption is enforced and there is no fallback to unencrypted DNS traffic.*

DNS requests from the local network should be encrypted via TLS before being sent to an upstream DNS server. This can be configured in the router under `Internet/Zugangsart/DNS-Server`. Here are some as an example:
```